scripts/api_lint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+npx @redocly/openapi-cli lint api/doc/openapi.yaml --skip-rule no-empty-servers
```

Comment:
you might need to make sure npx is installed.

Reply:
It probably is installed with npm or deno by dev_setup.sh, I have it local, so guess it works on build too.

The lint passed, so it is good :)